app/assets/stylesheets/views/confirm-email.scss

```diff
@@ -17,10 +17,10 @@
       padding: 1em 1em 0;
     }
   }
+}
 
-  .forem-background {
-    @media (max-width: $breakpoint-m) {
-      display: none;
-    }
+.forem-background {
+  @media (max-width: $breakpoint-m) {
```

Comment:
This background seems to be coming in the way of field in smaller screens. I need to find a more elegant way to implement the background. I hid it for now and will tackle in a separate PR.

Reply:
I wonder if instead of adding the inline SVG tag, using `style="background-image: url(<% path_to_asset %>);"` would help with this 🤔

Reply:
I tried that as well and I was having some issues with it too, mostly because the image is not the full width of the viewport and needs to be specifically placed on the corner, but honestly I didn't play with it long enough, I could try a couple of more options so I'll look into it some more after this PR. It needs to be applied to a couple of other pages as well so I'll invest some time into getting them all working properly.

Reply:
aaah yeah, bit of experimentation required! `background-position` and `background-size` might help - it should let you set it to stay the original size vs distorting, but yeah I'm sure it will take a bit of faff to get it just right for all the screen sizes. CSS eh 😆